harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/localfilesystem/LocalFsHarvesterFileVisitor.java

```diff
@@ -69,7 +69,7 @@
  * @author Jesse on 11/6/2014.
  */
 class LocalFsHarvesterFileVisitor extends SimpleFileVisitor<Path> {
-    private static final Logger LOGGER = LoggerFactory.getLogger(Geonet.HARVESTER);
+    private Logger LOGGER = LoggerFactory.getLogger(Geonet.HARVESTER);
```

Comment:
why not static final anymore ? because there can have several different instances which require their own loggers ?

Reply:
Not really static, but definitively not final ?

cf. https://github.com/georchestra/geonetwork/pull/140/files/be50afa452120c091e88a52fab109e65cac949fa#diff-72068ae957d6c5106543d93e2bb2ed79R111-R112

![harvester_logs](https://user-images.githubusercontent.com/18677090/81698462-4d23fa80-9466-11ea-9ef8-0be50a9b0836.png)